src/components/connectAdvanced.js

```diff
@@ -217,7 +243,7 @@ export default function connectAdvanced(
       }
 
       onStateChange() {
```

Comment:
If all this does is call `updateChildPropsFromReduxStore()`, we could just pass that to the `Subscription` instead.